docs/installation.rst

```diff
@@ -13,6 +13,20 @@ Requirements
 - The ``--container-runtime`` option must correspond to the containerization tool you use.
 - ``ansible-builder`` version ``3.x`` requires Python ``3.9`` or higher.
 
+.. note::
+
+   The **recommended** approach to install ``ansible-builder`` is using the
+   ``ansible-dev-tools`` package.
+   `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_
+   aims to streamline the setup and usage of several tools needed in order to
+   create `Ansible <https://www.ansible.com>`_ content. ADT combines critical Ansible
+   development packages into a unified Python package.
+
+   .. code-block:: shell
+
+      # This also installs ansible-core if it is not already installed
+      $ pip3 install ansible-dev-tools
```

Comment:
It might make sense to also fix the other 2 examples of using `pip`, and use the same executable `pip3`, which seems to be the default one (the simple one might be missing on some platforms), same applies to using python.

Reply:
updated